tests/e2e/helm-chart/chainsaw-test.yaml

```diff
@@ -25,14 +25,12 @@ spec:
     try:
     - script:
         content: |
-          git clone https://github.com/odigos-io/odigos-charts.git /tmp/odigos-charts
           # Retry to avoid flakiness (due to CRD race conditions in helm). Timeout after 60s.
-          while ! helm upgrade --install odigos /tmp/odigos-charts/charts/odigos --create-namespace --namespace odigos-test-ns --set image.tag=e2e-test; do
+          while ! helm upgrade --install odigos $GITHUB_WORKSPACE/helm/odigos --create-namespace --namespace odigos-test-ns --set image.tag=e2e-test; do
```

Comment:
nit: I used `/tmp` to easily be able to run chainsaw tests locally outside Github Actions

Reply:
I understand that.

We need to somehow find the project root. Chainsaw just changes the current working directory and, outside of GHA, I don't even know how to retrieve it.

Do you know a way to reference project root in chainsaw tests that is not GHA-specific?

Reply:
Changed in https://github.com/odigos-io/odigos/pull/1429/commits/c44935bec2823654ebc87fd01d11c2b6fcac5c4e